src/main/java/io/github/moulberry/notenoughupdates/listener/RenderListener.java

```diff
@@ -459,9 +460,24 @@ public void onGuiScreenDrawPre(GuiScreenEvent.DrawScreenEvent.Pre event) {
 		}
 	}
 
+	private static final String[] dungeonMenus =
+		{
+			"Spirit Leap",
+			"Revive A Teammate",
+			"Click in order!",
+			"What starts with",
+			"Select all the",
+			"Click the button on time!",
+			"Correct all the panes!",
+			"Change all to same color!"
+		};
+	private static final List<String> dungeonMenuList = Arrays.asList(dungeonMenus);
```

Comment:
You would typically use a set here instead of a list: 
```suggestion
	private static final Set<String> dungeonMenuList = new HashSet<>(Arrays.asList(dungeonMenus));
```